Gemfile.lock

```diff
@@ -22,26 +15,12 @@ GEM
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
     rspec-support (3.9.3)
-    rubocop (0.89.1)
-      parallel (~> 1.10)
-      parser (>= 2.7.1.1)
-      rainbow (>= 2.2.2, < 4.0)
-      regexp_parser (>= 1.7)
-      rexml
-      rubocop-ast (>= 0.3.0, < 1.0)
-      ruby-progressbar (~> 1.7)
-      unicode-display_width (>= 1.4.0, < 2.0)
-    rubocop-ast (0.3.0)
-      parser (>= 2.7.1.4)
-    ruby-progressbar (1.10.1)
-    unicode-display_width (1.7.0)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
   rspec
-  rubocop
```

Comment:
Why did you remove rubocop? We use Rubocop as a linter and code formatter and this helps us to improve our code quality and readability.

Reply:
I haven't touched Gemfile.lock. Since rubocop is not defined as a dependency in Gemfile it was automatically removed from the file when I did "bundle install".